README.md

```diff
@@ -89,6 +89,15 @@ $ appknox files 4
 ----  ------  ---------  --------------
    6  MFVA            1               6
    7  MFVA            1               6
+
```

Comment:
Perhaps we should remove these examples from here? Quick-start should be as simple as possible and it is unlikely normal users will have multiple accounts with Appknox and that they'll use profiles...